source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -110,442 +110,528 @@ volume and a supporting
       Ensure all Persistent Volumes provisioned to support the MinIO Tenant 
       use this storage class.
 
-Procedure (Operator Console)
-----------------------------
+.. _minio-k8s-deploy-minio-tenant-commandline:
 
-1) Access the MinIO Operator Console
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Procedure
+---------
 
-Use the :mc-cmd:`kubectl minio proxy` command to temporarily forward 
-traffic between the local host machine and the MinIO Operator Console:
+.. tab-set::
```

Comment:
So I'm looking at this, and I'm a little unsure if we should use tabs for these procedures:

1. We lose the header anchors for individual steps, and along with that the ability to link to specific sections
2. We imply that Operator Console = CLI in functionality, which is not the case. The Operator Console has far more flexibility
3. Eventually we need to add `Kustomize` or something for very advanced users, which I guess is not a big deal having three tabs.

The gain in not scrolling is kind of nice - but we have a floating right TOC, so a user could click to whatever H1 they want.

Thus my proposal - we should have three H1s on this page:

- Deploy MinIO Tenant using Operator Console
- Deploy MinIO Tenant using `kubectl minio`
- (Future) Deploy MinIO Tenant using `Kustomize`

Each section can have it's own pre-requisites (if necessary).  That also lets us describe specific behaviors cleanly (e.g. w/ `kubectl minio` TLS is always enabled)

LMK your thoughts.

Reply:
:thinking: there should be a way to get that scroll to work...though to your point if we have that many headings in the right TOC something else needs to get fixed up.